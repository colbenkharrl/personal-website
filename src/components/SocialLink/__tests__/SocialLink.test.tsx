import { render, screen } from '@testing-library/react';
import * as stories from '../__stories__/SocialLink.stories';
import { composeStories } from '@storybook/react';

const { Basic } = composeStories(stories);

it('SocialLink matches snapshot (regression test)', () => {
  render(<Basic data-testid="SocialLink" />);

  expect(screen.getByTestId('SocialLink')).toMatchSnapshot(
    'Full component snapshot.',
  );
});
