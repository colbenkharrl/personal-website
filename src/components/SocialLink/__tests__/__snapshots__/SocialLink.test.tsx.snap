// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialLink matches snapshot (regression test): Full component snapshot. 1`] = `
<a
  class="group -m-1 p-1"
  data-testid="SocialLink"
  href="https://strava.com/"
>
  <svg
    aria-hidden="true"
    class="h-6 w-6 fill-zinc-500 transition group-hover:fill-amber-600 dark:fill-zinc-400 dark:group-hover:fill-amber-400"
    viewBox="0 0 24 24"
  >
    <path
      d="M15.387 17.944l-2.089-4.116h-3.065L15.387 24l5.15-10.172h-3.066m-7.008-5.599l2.836 5.598h4.172L10.463 0l-7 13.828h4.169"
    />
  </svg>
</a>
`;
